src/api/audioApi.ts

```diff
@@ -28,48 +32,36 @@ export async function fetchPodcastsPage(
   context: Context,
   pageSize: number,
   page: number,
-): Promise<GQLAudioSearch> {
+): Promise<IAudioSummarySearchResult> {
   const response = await fetch(
     `/audio-api/v1/audio/?page-size=${pageSize}&page=${page}&audio-type=podcast`,
     context,
   );
 
-  const audioSearch: GQLAudioSearch = await resolveJson(response);
-  const results = await Promise.all(
-    audioSearch.results.map(audio => fetchPodcast(context, audio.id)),
-  );
-
-  return {
-    ...audioSearch,
-    results,
-  };
+  const audioSearch: IAudioSummarySearchResult = await resolveJson(response);
+  return audioSearch;
 }
 
 export async function fetchPodcastSeries(
   context: Context,
   podcastId: number,
-): Promise<GQLPodcastSeries> {
+): Promise<IAudioMetaInformation> {
   const response = await fetch(`/audio-api/v1/series/${podcastId}`, context);
-  const series: GQLPodcastSeries = await resolveJson(response);
-
-  const episodes = await Promise.all(
-    series.episodes.map(audio => fetchPodcast(context, audio.id)),
-  );
-
-  return { ...series, episodes };
+  const series: IAudioMetaInformation = await resolveJson(response);
+  return series;
```

Comment:
Her kunne du kanskje forenkla litt.
```suggestion
  return await resolveJson(response);
```